Content.Shared/SS220/CrayonRechargeable/CrayonRechargeableComponent.cs

```diff
@@ -0,0 +1,14 @@
+namespace Content.Shared.SS220.CrayonRechargeable
+{
+    [RegisterComponent]
+    public sealed partial class CrayonRechargeableComponent : Component
+    {
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
+        public int ChargesPerWait { get; set; } = 1;
+
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
+        public float WaitingForCharge { get; set; } = 2.3f;
```

Comment:
Для сериализуемых полей также можно использовать `TimeSpan` если речь идёт про время (сериализуется в виде количества секунд).
```suggestion
        public TimeSpan WaitingForCharge { get; set; } = TimeSpan.FromSeconds(2.3f);
```